src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,224 @@
+  const facility = facilityData?.[facilityId];
+
+  const isCC = appointment?.vaos.isCommunityCare;
+  const isExpressCare = appointment?.vaos.isExpressCare;
+  const isCommunityCare = appointment?.vaos.isCommunityCare;
+  const isVideo = isVideoAppointment(appointment);
+  const isInPersonVAAppointment = !isVideo && !isCommunityCare;
+
+  const header = 'VA Appointment';
+  const showInstructions =
+    isCommunityCare ||
+    (isInPersonVAAppointment &&
+      PURPOSE_TEXT.some(purpose =>
+        appointment?.comment?.startsWith(purpose.short),
+      ));
+
+  return (
+    <div>
+      <div className="vads-u-display--block vads-u-padding-y--2p5">
+        ‹ <Link to="/">Manage appointments</Link>
+      </div>
+
+      <h1>
+        <AppointmentDateTime
+          appointmentDate={moment.parseZone(appointment.start)}
+          timezone={appointment.vaos.timeZone}
+          facilityId={getVARFacilityId(appointment)}
+        />
+      </h1>
+
+      {!!facility &&
+        !isCC &&
+        !isExpressCare && (
+          <>
+            <div className="vads-u-font-size--sm vads-u-font-weight--normal vads-u-font-family--sans">
+              <span>
+                <b>{header}</b>
+              </span>
+            </div>
+
+            <VAFacilityLocation
+              facility={facility}
+              facilityName={facility?.name}
+              facilityId={parseFakeFHIRId(facilityId)}
+              isV2
+              clinicFriendlyName={appointment.participant[0].actor.display}
+            />
+
+            {showInstructions &&
+              isInPersonVAAppointment && (
+                <div className="vads-u-margin-top--3 vaos-appts__block-label">
+                  <AppointmentInstructions
+                    instructions={appointment.comment}
+                    isHomepageRefresh
+                  />
+                </div>
+              )}
+
+            <div className="vads-u-margin-top--3 vaos-appts__block-label">
+              <i
+                aria-hidden="true"
+                className="far fa-calendar vads-u-margin-right--1"
+              />
+              <AddToCalendar
+                summary={`${header}`}
+                description={`instructionText`}
+                location={location}
+                duration={appointment.minutesDuration}
+                startDateTime={moment.parseZone(appointment.start)}
+              />
+            </div>
+
+            <div className="vads-u-margin-top--2 vaos-appts__block-label">
+              <i
+                aria-hidden="true"
+                className="fas fa-print vads-u-margin-right--1"
+              />
+              <a href="#">Print</a>
+            </div>
+
+            <div className="vads-u-margin-top--2 vaos-appts__block-label">
+              <i
+                aria-hidden="true"
+                className="fas fa-clock vads-u-margin-right--1"
+              />
+              <a href="#">Reschedule</a>
```

Comment:
Needs to be either a full URL in the href or a button

Reply:
This functionality is in a later ticket, it's a placeholder for now